### DISCUSSION TITLE
Autotranslation on Google translate V1 API works slow.

### DISCUSSION THREAD

**Discussion by Name2889** [SubtitleEdit/subtitleedit#7660] [General]
Thanks a lot for this program. I have been using it for a long time, mainly to translate subtitles. 
1. Something changed in the last 4.0.2 version, and autotranslation became too slow. Before it took about 1-2 sec, now it takes about 25sec to translate. I translate mainly series of 20-25min videos. So not too many words.
2. After translation done, I save it to another folder, not the folder original subtitles were located at. So in 4.0.2 version, it does not remember folder for translated subtitles. It tries to save it in initial folder, where original subtitles were saved. Before it worked well, i.e. it remembered where to save translated subtitles.
3. I tried to use a new ChatGPT autotranslation as well. But it returned error, like "Too many requests"...
I would very appreciate if 1-2 would be fixed. 

Thank you.
Best regards,

**Comment by Name2889** [comment]
Rolled back to 4.0.1.  Works great.

**Comment by niksedk** [comment]
Note that SE 4.02 will give better results for texts spanning more than one line!

I will try to see if I can make it a tiny bit faster....
